### DISCUSSION TITLE
Compiler warnings - should they be addressed?

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4403] [General]
Is this a concern at all?
![image](https://github.com/xLightsSequencer/xLights/assets/4643499/78d67be9-c458-4893-a928-1a86720f7cb4)

**Comment by keithsw1111** [comment]
While we fixed a lot of these fixing some breaks code if you suppress it the wrong way. Generally unless you know for sure the value can never be less than zero then you are best off casting the unsigned to signed rather than vice versa. That said know king if it can be less than zero is often quite difficult to determine.

On 14 Mar 2024, at 12:25 am, Daryl ***@***.***> wrote:

﻿

Is this a concern at all?
image.png (view on web)<https://github.com/xLightsSequencer/xLights/assets/4643499/78d67be9-c458-4893-a928-1a86720f7cb4>
